addons/hide-show-context-menu/addon.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Hide sprites via context menu",
+  "description": "Adds an option to a sprite's right click menu to hide or show it.",
+  "tags": ["editor"],
+  "versionAdded": "1.36.0",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"]
+    }
+  ],
+  "credits": [
+    {
+      "name": "mybearworld",
+      "link": "https://scratch.mit.edu/users/mybearworld"
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true
```

Comment:
Dynamic disable isn't handled right now.

Reply:
The function seems to handle that too.